esrally/tracker/tracker.py

```diff
@@ -74,16 +74,19 @@ def create_track(cfg):
     track_name = cfg.opts("track", "track.name")
     indices = cfg.opts("generator", "indices")
     root_path = cfg.opts("generator", "output.path")
-    target_hosts = cfg.opts("client", "hosts")
-    client_options = cfg.opts("client", "options")
+    target_hosts = cfg.opts("client", "hosts").default
+    client_options = cfg.opts("client", "options").default
```

Comment:
What is this change doing?

Reply:
Just a shorthand way of getting the same values with:
```python
target_hosts.all_hosts[opts.TargetHosts.DEFAULT]
client_options.all_client_options[opts.TargetHosts.DEFAULT]
```